misk/src/main/kotlin/misk/concurrent/RealExecutorServiceFactory.kt

```diff
@@ -62,31 +66,51 @@ internal class RealExecutorServiceFactory @Inject constructor(
     awaitAllTerminated.start()
   }
 
+  // The traceWithActiveSpanOnly: false flag below means we will create a new root span if there is
```

Comment:
what is the problem with "too many root spans"? just potentially more for datadog to index? probably fine since we're sampling.

honestly it's good to surface them so we can attach them.

Reply:
Yeah I agree